.jenkins

```diff
@@ -345,7 +345,7 @@ pipeline {
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               ../gnu_generate_makefile.bash \
                                 --with-options=compiler_warnings \
-                                --cxxflags="-Werror -Werror all-warnings" \
+                                --cxxflags="-Werror -Werror all-warnings -Xcudafe --diag_suppress=20208" \
```

Comment:
I needed to move pragma push into the `Complex.hpp` to avoid seeing the warning. At this point, it seems to be better to just disable the warning for this build in the compiler flags.